src/test/java/org/reaktivity/specification/socks/internal/SocksFunctionsTest.java

```diff
@@ -542,7 +542,7 @@ public void shouldNotBuildBeginExWithInvalidIpv6Address() throws Exception
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void shouldNotBuildBeginExWithInvalidIpv6Address1() throws Exception
+    public void shouldNotBuildBeginExWithInvalidIpv6AddressWithInvalidGroups() throws Exception
     {
         byte[] bytes = SocksFunctions.beginEx()
```

Comment:
Please remove all of these unused assignments for negative tests that throw an exception, the call itself is sufficient to throw the exception in each case.

Reply:
If I move all negative tests, the code coverage will drop, like the `if` case.

Reply:
I'm not asking you to remove the negative tests, just the unused assignment above.
So the code would begin directly with `SocksFunctions.beginEx()`, not `byte[] bytes = SocksFunctions.beginEx()` in each case.

Reply:
I see, thanks.